### DISCUSSION TITLE
Cannot estimate gas ERROR, on portal and wormhole

### DISCUSSION THREAD

**Discussion by WCrohare6** [wormhole-foundation/wormhole#2484] [Problems / bugs] [answered]
Hi, I'll detail lall the steps I made and show you where the problem is, I'd read all the documentation but didn't find my error.

First I use app.atlasdex.finance to exchange 1000usdt from avalanche to polygon. But it didn't work ...

So I use https://www.portalbridge.com/#/redeem to redeem them, so it take to: https://www.portalbridge.com/#/transfer and ask me to change my blockchain to polygon
<img width="974" alt="imagen" src="https://user-images.githubusercontent.com/9695759/223615109-947a15e6-7234-4864-96ea-c541157995ad.png">
 I do that and try to redeem.... but the follow error appears:

<img width="1619" alt="imagen" src="https://user-images.githubusercontent.com/9695759/223615404-8acc2089-3bfa-4ad4-8b86-76a23d0caf3a.png">

`cannot estimate gas; transaction may fail or may require manual gas limit [ See: https://links.ethers.org/v5-errors-UNPREDICTABLE_GAS_LIMIT ] (reason="execution reverted: invalid sender", method="estimateGas", transaction={"from":"0xad21D72eb341aa6f9681e69EF714Cd....","to":"0x5a58505a96D1dbf8dF91cB21B...","data":"0xc687851900000..........","accessList":null}, error={"code":-32603,"message":"Internal JSON-RPC error.","data":{"code":3,"message":"execution reverted: invalid sender","data":"0x08c379a0000000000000..."}}, code=UNPREDICTABLE_GAS_LIMIT, version=providers/5.7.2)`

please I need help, I ask on discord and nobody answer

**Comment by evan-gray** [comment, marked as answer]
Hi @WCrohare6 this transfer appears to be a contract controlled transfer, which can only be redeemed by the designated contract (likely something related to Atlas, considering you used their app). Please reach out to their team for support but feel free to post back here with what you learned to help future Atlas users who may come here with the same issue.

**Comment by aadam-10** [comment]
closing the thread as answered. @WCrohare6 please feel free to drop a reply here in case you received information from Atlas.

**Comment by WCrohare6** [comment]
Yes everything is okey, thank you

El El mar, 21 de mar. de 2023 a la(s) 11:30, Adam A. <
***@***.***> escribió:

> closing the thread as answered. @WCrohare6 <https://github.com/WCrohare6>
> please feel free to drop a reply here in case you received information from
> Atlas.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/wormhole-foundation/wormhole/discussions/2484#discussioncomment-5381999>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACJ7EDZCXU6F3VBSYUYQMGDW5G3PDANCNFSM6AAAAAAWCQZ67M>
> .
> You are receiving this because you were mentioned.Message ID:
> <wormhole-foundation/wormhole/repo-discussions/2484/comments/5381999@
> github.com>
>
-- 

Wanda Croharé
